ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Services/UserService.cs

```diff
@@ -28,5 +29,30 @@ public Task<bool> ExistsAsync(Guid id, CancellationToken cancellationToken)
             cancellationToken.ThrowIfCancellationRequested();
             return _repository.ExistsAsync(id, cancellationToken);
         }
+
+        public Task<string> GetNameAsync(Guid id, CancellationToken cancellationToken)
+        {
+            cancellationToken.ThrowIfCancellationRequested();
+            return _repository.GetNameAsync(id, cancellationToken);
+        }
+
+        public Task<string> GetEmailAsync(Guid id, CancellationToken cancellationToken)
+        {
+            cancellationToken.ThrowIfCancellationRequested();
+            return _repository.GetEmailAsync(id, cancellationToken);
+        }
+
+        public async Task<List<Message>> WeeklyApproverEmailAsync(CancellationToken cancellationToken)
+        {
+            cancellationToken.ThrowIfCancellationRequested();
+            List<Message> messages = new List<Message>();
+            var emailList = await _repository.WeeklyApproverEmailAsync(cancellationToken);
+            foreach (var email in emailList)
+            {
+                messages.Add(new Message(new string[] { email }, "Weekly Approver Reminder", "Please remember to submit your approval tables today. Thank you"));
```

Comment:
reminder should be for all users, not only approvers, everyone should fill time sheets